docs/deployment/setup_gardener.md

```diff
@@ -1,44 +1,412 @@
+
+In this example, the runtime cluster is also used to host the very first `gardenlet`, which will register this cluster as a `Seed` as well. The `gardenlet` can be deployed to another Kubernetes cluster (recommended for productive setups). Here, using the runtime cluster keeps the amount of unmanaged clusters at a bare minimum.
+
+### DNS Zone
+
+The virtual Garden API endpoint (`kube-apiserver`) is exposed through a LoadBalancer `Service` and its IP address is made discoverable through DNS. Hence, a zone where an `A` record can be created is required.
+
+Additionally, Gardener advertises `kube-apiservers` of `Shoots` through two DNS records, which requires at least one DNS zone and credentials to interact with it through automation.
+
+Assuming the base domain for a Gardener landscape is `gardener.crazy-botany.cloud`, the following example shows a possible way to structure DNS records:
+
+| Component                | General                                            | Example                                              |
+|:-------------------------|:---------------------------------------------------|:-----------------------------------------------------|
+| Gardener Base            | gardener.\<showroom-base-domain\>                  | gardener.crazy-botany.cloud                          |
+| Virtual Garden           | api.garden.\<gardener-base-domain\>                | api.garden.gardener.crazy-botany.cloud               |
+| Gardener Ingress         | ingress.garden\<gardener-base-domain\>             | ingress.garden.gardener.crazy-botany.cloud           |
+| Gardener Dashboard       | dashboard.ingress.\<gardener-base-domain\>         | dashboard.ingress.garden.gardener.crazy-botany.cloud |
+| Shoot Internal           | internal.\<gardener-base-domain\>                  | internal.gardener.crazy-botany.cloud                 |
+| Shoot (default) External | shoot.\<gardener-base-domain\>                     | shoot.gardener.crazy-botany.cloud                    |
+| Unmanaged Seed Ingress   | ingress.soil.\<gardener-base-domain\>              | ingress.soil.gardener.crazy-botany.cloud             |
+| Managed Seed             | \<seed-name>.seed.\<gardener-base-domain\>         | cc-ha.seed.gardener.crazy-botany.cloud               |
+| Managed Seed Ingress     | ingress.\<seed-name>.seed.\<gardener-base-domain\> | ingress.cc-ha.seed.gardener.crazy-botany.cloud       |
+
+Reference documentation:
+
+- [Gardener Configuration](https://github.com/gardener/gardener/blob/master/docs/operations/configuration.md#system-configuration)
+- [DNS records](https://github.com/gardener/gardener/blob/master/docs/extensions/resources/dnsrecord.md)
+
+### Backup Bucket
+
+The virtual Garden cluster's `etcd` is governed by [etcd-druid](https://github.com/gardener/etcd-druid). Backups are created continuously and stored to a supported object store. Credentials to manage an object store on the chosen infrastructure are required.
```

Comment:
If I am not mistaken, etcd backup is optional, but recommended for productive installations of Gardener.

Reply:
Thanks for the hint. Addressed with a37e8e4b077f2b6ae111606124d82f0448377565